File: Module 2 - Loops (Iteration)/While Loop/ChallengeProblem/Passwordverify.py
""" Password Verification
Keep asking the user for a password until they enter:
admin123
Then display:
Access Granted """

newpassword = input("Create Password: ")

attempts = 0

while attempts < 3:
    password = input("Enter Password: ")

    if password == newpassword:
        print("Access Granted")
        break

    print("Password Incorrect.")
    attempts += 1

if attempts == 3:
    print("Access Denied")

File: Module 2 - Loops (Iteration)/While Loop/ChallengeProblem/PasswordVerify.cpp
/* Password Verification
Keep asking the user for a password until they enter: it in 3 attempt
admin123
Then display:
Access Granted */



#include <iostream>
using namespace std;

int main()
{
    string newPassword, password;
    int attempts = 0;

    cout << "Create a Password: ";
    cin >> newPassword;

    while(attempts < 3)
    {
        cout << "Enter Password: ";
        cin >> password;

        if(password == newPassword)
        {
            cout << "Access Granted";
            return 0;
        }

        attempts++;
        cout << "Password Incorrect.\n";
    }

    cout << "Access Denied";
}